lib/Test/Pod/CoverageChange.pm

```diff
@@ -0,0 +1,250 @@
+
+ Test::Pod::CoverageChange::pod_coverage_syntax_ok('lib', {
+     Module::With::3::expected::naked::subs              => 3,
+     AnotherModule::With::10::expected::naked::subs      => 10,
+     YetAnotherModule::With::1::expected::naked::subs    => 1,
+     YetAnotherModule::With::5::expected::naked::subs    => 5,
+ }, [
+     We::Ignore::ThisModule,
+     We::Also::Ignore::This::Module
+ ]);
+
+=head1 DESCRIPTION
+
+C<Test::Pod::CoverageChange> is a helper combining L<Test::Pod::Coverage> and
+L<Pod::Checker> to test for both POD coverage and syntax changes for a module
+distribution at once, via a single call to L</pod_coverage_syntax_ok>.
+
+Possible results
+
+=over 4
+
+=item * B<passes> if the file has no POD syntax or coverage error.
+
+=item * B<fails> if latest changes increased/decreased numbers of naked sub for the packages that have allowed naked subs.
+
+=item * B<fails> if a package allowed to have naked subs has 100% POD coverage.
+
+=item * B<fails> if a file in a given path has POD syntax error or has no POD.
+
+=back
+
+Ignores every package named from C<$ignored_package>.
+
+=cut
+
+use Test::More;
+use Pod::Checker;
+use Pod::Coverage;
+use File::Find::Rule;
+use Test::Pod::Coverage;
+use Module::Path qw(module_path);
+use List::Util qw(any);
+
+use constant {
+    POD_SYNTAX_IS_OK => 0,
+    FILE_HAS_NO_POD  => -1,
+};
+
+use Exporter qw(import export_to_level);
+our @EXPORT_OK = qw(pod_coverage_syntax_ok);
+
+=head2 pod_coverage_syntax_ok
+
+Checks all the modules under a given directory against POD coverage and POD syntax
+
+=over 4
+
+=item * C<$path> - path or arrayref of directories to check (recursively)
+
+example: ['lib', 'other directory'] | 'lib'
+
+=item * C<$allowed_naked_packages> - hashref of number of allowed naked subs, keyed by package name (optional)
+
+example: {Package1 => 2, Package2 => 1, Package3 => 10}
+
+=item * C<$ignored_packages> - arrayref of packages that will be ignored in the checks (optional)
+
+example: ['MyPackage1', 'MyPackage2', 'MyPackage3']
+
+=back
+
+=cut
+
+sub pod_coverage_syntax_ok {
+    my $path                   = shift;
+    my $allowed_naked_packages = shift // {};
+    my $ignored_packages       = shift // [];
+
+    $path             = [$path]             unless ref $path eq 'ARRAY';
+    $ignored_packages = [$ignored_packages] unless ref $ignored_packages eq 'ARRAY';
+
+    check_pod_coverage($path, $allowed_naked_packages, $ignored_packages);
+    check_pod_syntax($path, $ignored_packages);
+}
+
+=head2 check_pod_coverage
+
+Checks POD coverage for all the modules that exists under a given directory.
+Passes the C<$allowed_naked_packages> to L<Test::Pod::CoverageChange/check_allowed_naked_packages>.
+Ignores the packages in the C<$ignored_packages> parameter.
+
+=over 4
+
+=item * C<$path> - path or arrayref of directories to check (recursively)
+
+=item * C<$allowed_naked_packages> - hashref of number of allowed naked subs, keyed by package name (optional)
```

Comment:
```suggestion
=item * C<$allowed_naked_packages> - a hashref includes the number of allowed naked subs, keyed by package name (optional)
```

Reply:
the grammar is wrong (should be "which includes"), but please keep the original